src/components/display.component.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+const Display = ({result}) => (
+    <div>
+        <h4>{result}</h4>
+    </div>
+)
+
```

Comment:
- You should set the type of the `result` prop as `String` here see [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/validateAsString.png)
  and the default value which will be `'0'` see [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/DefaultProp.png)